testing/db-testkit/src/main/kotlin/net/corda/db/testkit/DbUtils.kt

```diff
@@ -4,105 +4,143 @@ import com.typesafe.config.Config
-        return if (!port.isNullOrBlank()) {
-            val ds = createPostgresDataSource(dbUser, dbPassword, schemaName, createSchema, rewriteBatchedInserts)
+    ):EntityManagerConfiguration{
+        println(System.getProperty(MSSQL_PORT_PROPERTY))
+        return if(!isPostgres || !isMSSQL) {
+            val ds =
+                createDataSource(dbUser, dbPassword, schemaName, createSchema, rewriteBatchedInserts)
             DbEntityManagerConfiguration(ds, showSql, true, DdlManage.NONE)
         } else {
             logger.info("Using in-memory (HSQL) DB".emphasise())
-            TestInMemoryEntityManagerConfiguration(inMemoryDbName, showSql).also {
-                if (createSchema) {
+            TestInMemoryEntityManagerConfiguration(inMemoryDbName,showSql).also{
+                if(createSchema){
                     it.dataSource.connection.createSchema(schemaName)
                 }
             }
         }
     }
 
-    /**
-     * Creates Postgres [CloseableDataSource]
-     *
-     * @param dbUser DB user. If value is not provided, value of the system property "postgresUser" is used.
-     *               If system property is not set then value "postgress" is used.
-     * @param dbPassword DB password. If value is not provided, value of the system property "postgresPassword" is used.
-     *                   If system property is not set then value "password" is used
-     */
-    fun createPostgresDataSource(
-        dbUser: String? = null,
+
+    fun createDataSource(
+        dbUser:String? = null,
         dbPassword: String? = null,
         schemaName: String? = null,
         createSchema: Boolean = false,
         rewriteBatchedInserts: Boolean = false
     ): CloseableDataSource {
-        val port = System.getProperty(POSTGRES_PORT_PROPERTY)
-        val postgresDb = getPostgresDatabase()
-        val host = getPropertyNonBlank(POSTGRES_HOST_PROPERTY, "localhost")
-        var jdbcUrl = "jdbc:postgresql://$host:$port/$postgresDb"
-
-        val factory = PostgresDataSourceFactory()
-
         val user = dbUser ?: getAdminUser()
         val password = dbPassword ?: getAdminPassword()
-        if (!schemaName.isNullOrBlank()) {
-            if (createSchema) {
-                logger.info("Creating schema: $schemaName".emphasise())
-                factory.create(jdbcUrl, user, password, maximumPoolSize = 1).connection.createSchema(schemaName)
-            }
-            jdbcUrl = if (rewriteBatchedInserts) {
-                "$jdbcUrl?currentSchema=$schemaName&reWriteBatchedInserts=true"
-            } else {
-                "$jdbcUrl?currentSchema=$schemaName"
+        if (!isPostgres){
+            val port = System.getProperty(POSTGRES_PORT_PROPERTY)
+            val host = getPropertyNonBlank(POSTGRES_HOST_PROPERTY,"localhost")
+            val factory = PostgresDataSourceFactory()
+            val db = getDatabase()
+            var jdbcurl = "jdbc:postgresql://$host:$port/$db"
```

Comment:
```suggestion
            var jdbcUrl = "jdbc:postgresql://$host:$port/$db"
```

Use camel-case for `jdbcUrl`. Previously it was like this, so not sure why it was changed. There's a few instances of this,  they should all be corrected.